{
  "cveId": "CVE-2024-25132",
  "title": "Openshift-dedicated: hive: hibernation controller denial of service",
  "description": "A flaw was found in the Hive hibernation controller component of OpenShift Dedicated. The ClusterDeployment.hive.openshift.io/v1 resource can be created with the spec.installed field set to true, regardless of the installation status, and a positive timespan for the spec.hibernateAfter value. If a ClusterSync.hiveinternal.openshift.io/v1alpha1 resource is also created, the hive hibernation controller will enter the reconciliation loop leading to a panic when accessing a non-existing field in the ClusterDeploymentâ€™s status section, resulting in a denial of service.",
  "publishedDate": "2025-03-19T17:57:14.659000+00:00",
  "lastModifiedDate": "2025-05-22T16:48:01.908000+00:00",
  "severity": "MEDIUM",
  "cvssMetrics": [
    {
      "version": "3.1",
      "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L",
      "baseScore": 4.3,
      "baseSeverity": "MEDIUM"
    }
  ],
  "epss": {
    "score": 0.0005,
    "percentile": 16.215
  },
  "riskScore": 31,
  "exploitationStatus": "UNKNOWN",
  "affectedSystems": {
    "cpeMatches": [],
    "vendors": [],
    "products": []
  },
  "references": [
    {
      "url": "https://access.redhat.com/security/cve/CVE-2024-25132",
      "source": "",
      "tags": ["vdb-entry", "x_refsource_REDHAT"]
    },
    {
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2260371",
      "source": "RHBZ#2260371",
      "tags": ["issue-tracking", "x_refsource_REDHAT"]
    }
  ],
  "attackTechniques": [],
  "metadata": {
    "tags": ["CWE-400"],
    "requiresUserInteraction": null,
    "requiresPrivileges": null,
    "attackVector": null,
    "sources": [
      {
        "name": "CVEList",
        "url": "https://github.com/CVEProject/cvelistV5/blob/main/cves/2024/25xxx/CVE-2024-25132.json",
        "lastModified": "2025-05-22T16:48:01.908000+00:00"
      }
    ]
  }
}
